perun-openapi/openapi.yml

```diff
@@ -924,6 +924,14 @@ components:
           items:
             type: integer
 
+    #Map<Member, List<RichUser>>
+    MembersAndTheirSponsors:
+      type: object
+      properties:
+        member: { $ref: '#/components/schemas/Member' }
+        sponsors: { type: array, items: { $ref: '#/components/schemas/RichUser' }}
```

Comment:
I don't think that this will work since the returned json doesn't contains properties "member" and "sponsors", but is plain object, where keys are the member objects and their values are plain lists of sponsors. Maybe see the definition above or consult it with @martin-kuba.

Reply:
I created new class `MemberWithSponsors` and I used it as returned object so I completely changed `openapi` specification, too.